recipe/windows_cmake_install_bindir.patch

```diff
@@ -0,0 +1,13 @@
+diff --git CMakeLists.txt CMakeLists.txt
+index 3fd9d70..9ce3f67 100644
+--- CMakeLists.txt
++++ CMakeLists.txt
+@@ -398,7 +398,7 @@ endif ()
+ # installation directories
+ if (OS_WINDOWS)
+   set (RUNTIME_INSTALL_DIR Bin)
+-  set (LIBRARY_INSTALL_DIR Lib)
++  set (LIBRARY_INSTALL_DIR Bin)
```

Comment:
Actually we need to change the `install` command since it installs everything in `LIBRARY_INSTALL_DIR`. 

I think this should do it:
```
install(TARGETS <library_name>
        RUNTIME DESTINATION ${RUNTIME_INSTALL_DIR}
        LIBRARY DESTINATION ${LIBRARY_INSTALL_DIR}
        ARCHIVE DESTINATION ${LIBRARY_INSTALL_DIR})
```

see https://github.com/gflags/gflags/blob/v2.2.0/CMakeLists.txt#L504